Libraries/Text/Text.windows.js

```diff
@@ -1,576 +0,0 @@
-/**
- * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
```

Comment:
Ensure Text still works.

Reply:
Text still works as expected.

---
In reply to: [169649482](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169649482) [](ancestors = 169649482)